about.html

```diff
@@ -0,0 +1,117 @@
+                <div class="columns small-6 medium-6">
+                  <div id="logo">
+                      <p>Billboard Tracking System</p>
+                  </div>
+                </div>
+                  <div class="columns small-6 medium-6">
+                    <nav class="nav">
+
+                      <div class="hamburger-container">
+                        <ul class="hamburger">
+                          <li></li>
+                          <li></li>
+                          <li></li>
+                        </ul>
+                      </div>
+                    
+                      <ul id="menu">
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="contact.html">Contact</a></li>
+                      </ul>
+                    
+                    </nav>
+                  </div>
+            </div>
+
+            <script>
+              $(document).ready(function() {
+                //toggle menu
+                $(".hamburger-container").click(function() {
+                  $("#menu").slideToggle();
+                });
+
+                //to fix issue that toggle adds style(hides) to nav
+                $(window).resize(function() {
+                  if (window.innerWidth > 1024) {
+                    $("#menu").removeAttr("style");
+                  }
+                });
+
+                //icon animation
+                var topBar = $(".hamburger li:nth-child(1)"),
+                  middleBar = $(".hamburger li:nth-child(2)"),
+                  bottomBar = $(".hamburger li:nth-child(3)");
+
+                $(".hamburger-container").on("click", function() {
+                  if (middleBar.hasClass("rot-45deg")) {
+                    topBar.removeClass("rot45deg");
+                    middleBar.removeClass("rot-45deg");
+                    bottomBar.removeClass("hidden");
+                  } else {
+                    bottomBar.addClass("hidden");
+                    topBar.addClass("rot45deg");
+                    middleBar.addClass("rot-45deg");
+                  }
+                });
+              });
+
+              </script>
+          
+</header>
+<!-- End of Header Section -->
+    
+<!-- Start Main Conent Section -->
+<section id="content">
+    <div class="row">
+        <div class="columns medium-12" id="about">
+            <img class="img-center" alt="billboard" src="assets/images/billboard-icon.png">
+            <h3>Our Special Billboard Tracking System</h3>
+            <p>Our billboard tracking system allows business owners in the Philadlephpia area see which billboards are available
+                to rent, sizes, and locations.  In addition, our special application also provides a breakdown of local demographics, which
+                we believer is an important factor to take into account when considering an outdoor advertising campaigne.
```

Comment:
Couple of typos to fix:

line 101 = "believer" should be "believe"
line 101 = "campaigne" should be "campaign"